module/zfs/zfs_replay.c

```diff
@@ -419,9 +422,12 @@ zfs_replay_create_acl(void *arg1, void *arg2, boolean_t byteswap)
 	}
 
 bail:
-	if (error == 0 && zp != NULL)
+	if (error == 0 && zp != NULL) {
+#ifdef __FreeBSD__
+		VOP_UNLOCK1(ZTOV(zp));
```

Comment:
Do I understand correctly that `VOP_UNLOCK1` is a new FreeBSD wrapper added for this case?

Reply:
It's a wrapper around VOP_UNLOCK because it previously took two arguments and now takes one.